spinnman/utilities/locate_connected_machine_ip_address.py

```diff
@@ -53,8 +53,11 @@ def _ctrlc_handler(sig, frame):  # @UnusedVariable
         sys.exit()
 
     def _print_connected(ip_address, timestamp):
-        print(ip_address, "({})".format(
-            socket.gethostbyaddr(ip_address)[0]), "at", timestamp)
+        try:
+            hostname = socket.gethostbyaddr(ip_address)[0]
+        except Exception:
+            hostname = "Unknown host"
```

Comment:
I would use a string different than the one the IOS may give so it is easier to trace where it game from.

Reply:
Replaced with the removal of any message if the host cannot be resolved.  This is just a "helpful extra" but the IP address is the most important part.